frontend/src/common/queries/cellCards.ts

```diff
@@ -84,7 +83,7 @@ export function useCellCardQuery<T = CellCardResponse>(
 /* ========== ontology_tree ========== */
 export const USE_CELL_ONTOLOGY_TREE_QUERY = {
   entities: [ENTITIES.CELL_CARDS_CELL_ONTOLOGY_TREE],
-  id: "cell-explorer-cell-ontology-tree-query",
+  id: "cell-cards-cell-ontology-tree-query",
```

Comment:
renaming the entity strings to "cell-cards" instead of cell-explorer for consistency